### ISSUE TITLE
[Envoy] Blade Template -- No Newline Breaks Deployment

### ISSUE THREAD

**Issue by CWSpear** [laravel/framework#8152]
Using the code from the example in the documentation:

```php
@servers(['web' => '192.168.1.1'])

@task('deploy', ['on' => 'web'])
    cd site
    git pull origin {{ $branch }}
    php artisan migrate
@endtask
```

and running `envoy run deploy --pretend --branch="banana"`, I get the following output:

```
cd site
git pull origin banana    php artisan migrate
```

I expect:

```
cd site
git pull origin banana
php artisan migrate
```

(and indeed, that's the code it tries to execute if I don't `--pretend`, causing an error in the deployment)

This issue is mentioned in #4638 and @taylorotwell commented that he fixed it for Envoy, so I'm not sure if this is a regression or what? But I think it looks like it needs to be looked at again.

The current solution isn't terribly awful (changing the one line to `git pull origin {{ $branch . "\n" }}`, but it took me a long time to track down the first time, so it would be nice to be fixed or at least the docs updated to mention it.

**Comment by CWSpear** [comment]
If it matters, I am using Envoy standalone, i.e. my `composer.json`:

```json
{
    "require": {
        "laravel/envoy": "~1.0"
    }
}
```

I was getting errors on any version before `1.0.16`, but was able to reproduce this issue on every version between `1.0.16` and `1.0.21` (the latest at the time of this writing) except `1.0.20` which gave me an `no macro 'deploy'` error.

**Comment by JeroenNoten** [comment]
I can confirm this problem.

**Comment by makamo** [comment]
Just put ``` ; ``` after each line

**Comment by CWSpear** [comment]
The fix isn't difficult. Trying to diagnose why copying a script verbatim from the docs and having it not work is the bare minimum issue here.

**Comment by GrahamCampbell** [comment]
> Just put ; after each line

:+1

**Comment by CWSpear** [comment]
@GrahamCampbell Can we at least update the docs then? Copying a pasting a example from the docs and not having it work is very frustrating.

The following snippet found on http://laravel.com/docs/5.1/envoy, **does not work** as it currently is:

```bash
@servers(['web' => '192.168.1.1'])

@task('deploy', ['on' => 'web'])
    cd site
    git pull origin {{ $branch }}
    php artisan migrate
@endtask
```

No comment about needing a `;`, just arguably broken code, and if you argue it's not broken, then it's *definitely* a broken example. Very much not :+1: 

:cry:

**Comment by GrahamCampbell** [comment]
@JeroenNoten Appears to be prepping a PR. Thanks man. :)

**Comment by CWSpear** [comment]
Sweet. Now I can :+1:. More than just fixing the example, it may good to call it out explicitly.

**Comment by JeroenNoten** [comment]
No problem!

An explicit note may be nice indeed.

**Comment by CWSpear** [comment]
Thanks @JeroenNoten! I guess I *could* have done a PR, but wanted to talk about if it would be better to fix the code instead, and I was having a harder time figuring *that* one out.

**Comment by JeroenNoten** [comment]
Yep, for me the same, couldn't find an easy way to solve it. And because this issue is here already for a long time, I thought a fix in the documentation is a better (maybe temporary) solution. Hopefully there will be a fix in the code some time...

**Comment by taylorotwell** [comment]
This issue also existed in Blade and was fixed sometime last year so we probably just need a similar fix here.

**Comment by taylorotwell** [comment]
Fixed.

**Comment by taylorotwell** [comment]
This should be fixed in Envoy 1.0.25.
